packages/wallets/src/features/cfd/constants.tsx

```diff
@@ -121,9 +122,22 @@ export const CFD_PLATFORMS = {
     MT5: 'mt5',
 } as const;
 
+export const DESKTOP_PLATFORMS = {
+    LINUX: 'linux',
+    MACOS: 'macos',
+    WINDOWS: 'windows',
+} as const;
+
+export const MOBILE_PLATFORMS = {
+    ANDROID: 'android',
+    HAUWEI: 'huawei',
+    IOS: 'ios',
+} as const;
+
 export const MARKET_TYPE = {
     ALL: 'all',
     FINANCIAL: 'financial',
+    GAMING: 'gaming',
```

Comment:
market_type gaming and synthetic are the same, no need to add gaming as this is supposed to be removed from the code base